projects/dendrites/permutedMNIST/experiments/centroid.py

```diff
@@ -50,6 +50,11 @@ class CentroidFigure1BExperiment(CentroidFigure1B,
     pass
 
 
+class SICentroidExperiment(mixins.SynapticIntelligence,
```

Comment:
Maybe move these to a separate `prototype_si.py` experiment file? We should also have an experiment file for SI only.

Reply:
Done. Also, there is an experiment file for SI only, and that's `projects/vernon_continual/experiments/si.py`